meetings/2022-05-09.md

```diff
@@ -43,28 +46,124 @@
+    * SC: The community needs the authors to look at the tests and confirm that the test is checking the requirement.
+    * MJ: When we find differences in servers, we should be able to publish about that. Then we can write down what the process
+    * SC: test suite panel should not be creating issues about implementation problems.
+    * PE: What does point 3 practically mean?
+    * SC: Anyone can write tests. They write. If the requirement is part of spec A, then the writers of that spec need to look at the test and validate that the test is giving the requested output.
+    * PE: The focus is to acknowledge that the spec writers should be in the loop for validating that the test is doing the right thing for the requirement.
+    * SC: The proposal is to create an issue in bullet form that the author reviews the test. Need to have agreement on this point to have a charter.
+    * MJ: If we always need to include the authors for any reporting, and they don't, the test suite panel would not be allowed to publish anything.
+    * MJ: if our requirements require us to collaborate, but they don't, our charter prevents us from doing our work.
+    * MJ: Suggest: we continue to publish what we publish. cannot do that on solidproject.org because the website maintainer do not approve at this moment. Publish the information on another website, good resource for developers and maintainers. We ask solidproject.org if they want to merge our reports, but as long as they say no we have to publish it elsewhere until that time.
+    * SC: Number of issues on the reports.
+    * MJ: number of issues opened, but back-and-forth
+    * SC: This is the problem, because the tests are not authoritative. What kind of assurance can we give that the tests are reliable that they can look at. It currently is not reliable because there are disputes between different parties.
+    * MJ: This leads to spec discussion, which improves the
+    * SC: did the spec authors review all the tests?
+    * MJ: No, just the disputes.
+    * SC: We need the authors to validate the tests.
+    * MJ: Whenever we find a discrepancy between servers - if it is a problem on the server side, we let the server developer know. If the spec is unclear, we let the spec writers know.
+    * PE: N3 tests are not correct yet. But the numbers are already counted. It is known that the tests are not representative yet.
+    * MJ: there is a clear consensus that solid requires WAC. So at that point, we test for WAC.
+    * SC: unless the authors sign off on the tests, the point of the proposal is what the authors need.
+    * MJ: The test suite already has shown issues in the spec which have been resolved.
+    * SC: changes to solidproject.org go through a PR so that changes are documented.
+    * MJ: will report on solidservers.org for now, and hope that it can be merged into solidproject.org.
```

Comment:
Actioned this just now, see: https://solidservers.org/
Also [withdrew](https://github.com/solid/solidproject.org/pull/690/commits/ff3db83427c8928802c839e3b8f2b54fbb4c565a) request to link from https://solidproject.org to our work.
It's a sad situation to find ourselves in, but I'm confident that with Yvo's proposed way forward, documented in our charter and executed persistently, one day we'll get to 100% reviewed&approved by the solidproject.org webmasters team and by the spec team (I guess those two go hand-in-hand) and we can reopen our request to link from solidproject.org to solidservers.org, or even just merge the two sites into one, or whatever feels appropriate by that time.

Reply:
> Yvo's proposed way forward
Was this minuted - I had to leave and this may have come after that point

Reply:
I'm looking at the hackmd changelog, and the last snapshot was on 14:55 UTC. And, no, it wasn't minuted before the top of the hour.

Reply:
https://github.com/solid/test-suite-panel/pull/1/commits/7f4a73433a4f6aa28fbb7e6fb75de741f90f7315